### PR TITLE
Tests for scooter & order services.

### PR DESCRIPTION
**Pull request by avobl**

### CODE REVIEW

**Review by useralexey2000 — APPROVED** [review]
Look at conflicts and resolve them

File: repositories/account.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:generate mockgen -source=account.go -destination=../services/test/mock/mock_account.go -package=mock
```
Comment (DmytroTHR) [inline repositories/account.go]: I think we should use one directory for mocks.
Either /test/mock or just /mock (like in the next file)

File: services/account_test.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
+package services
+
```
Comment (DmytroTHR) [inline services/account_test.go:2]: Do we make separate test files for each service or one big test like Igor advised us yesterday?

**Review by DmytroTHR — APPROVED** [review]
resolve conflicts first (some of them may be because of me, sorry))

**Pull request merged**